src/adera/Shaders/PlumeShader.cpp

```diff
@@ -80,9 +80,9 @@ void PlumeShader::draw_plume(
     ACompDrawTransform const &drawTf    = rData.m_pDrawTf->get(ent);
     ACompExhaustPlume const &comp       = rData.m_pExaustPlumes->get(ent);
     PlumeEffectData const &effect       = *comp.m_effect;
-    Magnum::GL::Mesh &rMesh             = *rData.m_pMesh->get(ent).m_mesh;
+    Magnum::GL::Mesh &rMesh             = rData.m_pMeshGl->get(rData.m_pMeshId->get(ent));
```

Comment:
Not sure how exited I am with a "get()" function returning a reference.

Maybe do "at()" ?

Reply:
This is an entt::storage, it doesn't use exceptions and the `at` function does something entirely different. The entities there should always be valid unless something goes wrong; there's asserts in the get function for that.

Reply:
I wasn't really commenting on the possibility for exceptions.

I'm just used to thinking of "get()" always returning a pointer, that i find it odd that we're getting a reference back.

But i guess when in rome...